components/script/dom/servohtmlparser.rs

```diff
@@ -380,17 +375,15 @@ impl ServoHTMLParser {
             document: JS::from_ref(document),
         };
 
-        let tb = TreeBuilder::new(sink, TreeBuilderOpts {
-            ignore_missing_rules: true,
```

Comment:
This isn’t necessary anymore since all rules are implemented in html5ever’s tree builder now: https://github.com/servo/html5ever/commit/c40cf9a15ae4c1262062532fc98ae5320058d68d